README.md

```diff
@@ -56,6 +66,50 @@ This option allows you to disable it if you want:
 
 ```cordova run android ios -- --livereload --ghostMode=false```
 
+## Using it as an NPM package
+
+This codebase can also be used as an NPM package, making it easier to integrate in your custom workflows.
+Here's an example of how to use it:
+
+
+
+       var lr = require('cordova-plugin-livereload');
+       var cordova = require('cordova-lib');
+
+       //  Start LiveReload server
+       var projectRoot = '/home/omefire/Projects/mileage-tracker';
+       var platforms = ['android', 'ios'];
+
+       return lr.start(projectRoot, platforms, {
+           ghostMode: true,
+           ignore: 'build/**/*.*',
+           cb: function (event, file, lrHandle) {
+           
+               // After a file changes, first run `cordova prepare`, then reload.
+               cordova.raw.prepare().then(function () {
+                   var patcher = new lr.Patcher(projectRoot, platforms);
+                   return patcher.removeCSP();
+               }).then(function () {
+                   if (event === 'change') {
+                       return lrHandle.reloadFile(file);
```

Comment:
Would reloading the single changed file work in case of hrefs (lets say, in iframes) referencing this changed file?